demo-apps/cf-plugtest-client/pom.xml

```diff
@@ -36,6 +36,10 @@
 			<groupId>${project.groupId}</groupId>
 			<artifactId>cf-cli</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.californium</groupId>
```

Comment:
Not that important, but may be replaced by

`<groupId>${project.groupId}</groupId>`

Reply:
Now updated in commit 705e27c9636e950128fa814e04bb0460dafa2778